go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openfga/openfga
 
-go 1.22.4
+go 1.22.5
```

Comment:
If we set this to `1.22`, won't the latest available patch version always get retrieved so that would mean we wouldn't need to constantly bump the patch version all the time? 🤔

Reply:
But, IMHO, sometimes it can be a bit risky to do that, because there might be minor breaking changes, so I think, it's much better to use the specific version

Reply:
oh, I didn't know about that. TIL

thanks ❤️

Reply:
I tried writing just Go 1.22 but both VS Code and Goland automatically change it to include a minor version. I don't know why

Reply:
Then, it would be "recommended" practice (according to their devs at least).

Also, I think, including a specific version ensures that everyone who builds the project will use the same version of the dependency, which helps in avoiding issues caused by changes in dependencies.